php/test/decimal_to_precision.php

```diff
@@ -1,397 +0,0 @@
-<?php
-namespace ccxt;
-include_once (__DIR__.'/../../ccxt.php');
```

Comment:
this was a transpiled file already, why is being removed?

Reply:
it was just renamed into the same name as JS (which is more suitable name): https://github.com/ccxt/ccxt/pull/17075/files#diff-52b2da23b9d206589bfb46a82500340b9cbfb06b4ba3cd6e547e73a7d217a785R1784